src/turnkeyml/run/torchrt/runtime.py

```diff
@@ -58,7 +58,7 @@ def _setup(self) -> None:
             # First ensure we have the required version of Pytorch
             clean_torch_version = self.runtime_version.split("+")[0]
             if version.parse(clean_torch_version) < version.parse("2.0.0"):
-                exp.BenchmarkException(
```

Comment:
This was an unrelated bug: we were instantiating an exception without raising it :(

Reply:
Oh wow. Great catch!

Reply:
thank pylint!